engine/class_modules/sc_death_knight.cpp

```diff
@@ -4105,7 +4077,7 @@ struct army_of_the_dead_t : public death_knight_spell_t
 
     if ( is_precombat )
     {
-      double MIN_TIME = player -> base_gcd.total_seconds(); // the player's base unhasted gcd: 1.5s
+      double MIN_TIME = 1.5; // the player's base unhasted gcd: 1.5s
```

Comment:
We should continue to rely on the engine for this value instead of hard coding it.  While its unlikely the base gcd changes anytime soon, magic numbers should be avoided whenever possible.